fireant/slicer/queries/execution.py

```diff
@@ -93,17 +93,18 @@ def _do_fetch_data(query: str, database: Database):
         return pd.read_sql(query, connection, coerce_float=True, parse_dates=True)
 
 
-def _reduce_result_set(results: Iterable[pd.DataFrame],
-                       reference_groups,
-                       dimensions: Iterable[Dimension],
-                       share_dimensions: Dimension):
+def reduce_result_set(results: Iterable[pd.DataFrame],
+                      reference_groups,
+                      dimensions: Iterable[Dimension],
+                      share_dimensions: Iterable[Dimension]):
     """
     Reduces the result sets from individual queries into a single data frame. This effectively joins sets of references
     and concats the sets of totals.
 
     :param results: A list of data frame
     :param reference_groups: A list of groups of references (grouped by interval such as WoW, etc)
     :param dimensions: A list of dimensions, used for setting the index on the result data frame.
+    :param share_dimensions:
```

Comment:
Just noticed this docstring param is empty

Reply:
added a little blurb about it